config/application.rb

```diff
@@ -25,6 +25,17 @@ class Application < Rails::Application
     
     config.middleware.insert_after "Rails::Rack::Logger", "MiddlewareHealthcheck"
 
+    policies = [
+      #'default-src ' + ['\'self\''].join(' '),
+      'script-src-elem ' + ['\'self\'',
+        "\'unsafe-inline\'",
+        #"\'nonce-view012345\'",
```

Comment:
do you want to keep the 2 commented lines in here?

Reply:
second the question

Reply:
yes, but I should document what they are